### ISSUE TITLE
Revise main menu toggle for mobile for accessibility

### ISSUE THREAD

**Issue by paregorios** [isawnyu/isaw.web#244]
> role = button (ARIA), button (HTML5)
> The hamburger menu icon will need to be constructed as a button. An aria-label of “Main Menu Toggle” should be attributed. The toggle button needs visually hidden text content describing the purpose and/or current state of the button. 

Traceability: [Microassist Report 1 > Navigation Screen Reader Layout – Mobile > Landmarks](https://docs.google.com/document/d/10cfUFx-o_kSGBFKCe4MtL45cyc-c-Gwaz3Dxlrp8Qgg/edit#bookmark=id.jh9ul59ju21u)

**Comment by paregorios** [comment]
![image](https://user-images.githubusercontent.com/263285/36906337-48a7be76-1dfb-11e8-8896-d10e79455d55.png)
